### PR TITLE
Update Overview timeline with new statuses

### PR DESCRIPTION
**Pull request by dhaselhan**
* Update mapping of statuses and new HTML labels

### CODE REVIEW

File: alcs-frontend/src/app/features/application/overview/overview.component.ts [hunk 1]
```diff
@@ -162,61 +160,68 @@ export class OverviewComponent implements OnInit, OnDestroy {
...
+        if (status.status.code === SUBMISSION_STATUS.RECEIVED_BY_ALC) {
+          htmlText = 'Received All Items - <strong>Received by ALC</strong>';
+        }
+
+        if (status.status.code === SUBMISSION_STATUS.IN_PROGRESS) {
+          htmlText = 'Created - <strong>In Progress</strong>';
+        }
+
+        if (status.status.code === SUBMISSION_STATUS.SUBMITTED_TO_ALC_INCOMPLETE) {
+          htmlText = 'Acknowledge Incomplete - <strong>Submitted to ALC - Incomplete</strong>';
```
Comment (sandratoh) [inline alcs-frontend/src/app/features/application/overview/overview.component.ts:183]:
```suggestion
          htmlText = 'Acknowledged Incomplete - <strong>Submitted to ALC - Incomplete</strong>';
```

File: alcs-frontend/src/app/features/application/overview/overview.component.ts [hunk 2]
```diff
@@ -79,9 +80,6 @@ export class OverviewComponent implements OnInit, OnDestroy {
             this.decisionService.fetchByApplication(app.fileNumber).then((res) => {
               this.$decisions.next(res);
             });
-            this.reviewService.fetchReview(app.fileNumber).then((res) => {
-              this.$review.next(res);
-            });
```
Comment (sandratoh) [inline alcs-frontend/src/app/features/application/overview/overview.component.ts:84]: Is `reviewService` used elsewhere in the file or can we also remove its import and constructor?

File: services/apps/alcs/src/common/automapper/application-decision-v1.automapper.profile.ts [hunk 3]
```diff
@@ -160,6 +160,10 @@ export class ApplicationDecisionProfile extends AutomapperProfile {
         mapper,
         ApplicationDecisionMeeting,
         ApplicationDecisionMeetingDto,
+        forMember(
+          (a) => a.date,
+          mapFrom((ad) => ad.date.getTime()),
```
Comment (sandratoh) [inline services/apps/alcs/src/common/automapper/application-decision-v1.automapper.profile.ts:165]: Is this for changing db datetime to be readable by frontend

File: alcs-frontend/src/app/features/application/overview/overview.component.ts [hunk 4]
```diff
@@ -162,61 +160,68 @@ export class OverviewComponent implements OnInit, OnDestroy {
...
+          htmlText = 'Created - <strong>In Progress</strong>';
+        }
+
+        if (status.status.code === SUBMISSION_STATUS.SUBMITTED_TO_ALC_INCOMPLETE) {
+          htmlText = 'Acknowledge Incomplete - <strong>Submitted to ALC - Incomplete</strong>';
+        }
+
+        mappedEvents.push({
+          htmlText,
+          startDate: new Date(status.effectiveDate + status.status.weight),
```
Comment (sandratoh) [inline alcs-frontend/src/app/features/application/overview/overview.component.ts:188]: Is the weight coming from backend and similar concept as SORTING_ORDER?

File: services/apps/alcs/src/common/automapper/application-decision-v1.automapper.profile.ts (hunk 3 above)
Comment (dhaselhan) [reply services/apps/alcs/src/common/automapper/application-decision-v1.automapper.profile.ts:165]: It was sending the date as a string, but we should send dates as numbers to be consistent with the rest of the app.

File: alcs-frontend/src/app/features/application/overview/overview.component.ts (hunk 4 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/application/overview/overview.component.ts:188]: Yes! This is an easy way for me to sort them in the same order that the backend applies.

**Review by sandratoh — APPROVED** [review]
LGTM 👍

File: alcs-frontend/src/app/features/application/overview/overview.component.ts (hunk 2 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/application/overview/overview.component.ts:84]: Good catch, removed

**Pull request merged**